src/USER-PHONON/dynamical_matrix.cpp

```diff
@@ -0,0 +1,482 @@
+//
+// Created by charlie sievers on 6/21/18.
+//
+
+#include <mpi.h>
+#include <stdlib.h>
+#include <iostream>
```

Comment:
please avoid `<iostream>`. LAMMPS uses stdio from the c library, and mixing stdio with iostreams can lead to all kinds of problems across platforms.